src/Integration/Telemetry/TelemetryManager.cs

```diff
@@ -44,19 +45,25 @@ public sealed class TelemetryManager : ITelemetryManager, IDisposable
         private readonly ICurrentTimeProvider currentTimeProvider;
         private readonly IVsVersion vsVersion;
 
+        // Note: there isn't a MEF test for this constructor.Writing one causes other tests to fail.
```

Comment:
I think it would be better to create a Mef "test", mark it as Ignored, and paste that comment there. The comment doesn't really belong in product code.